t/bundle.t

```diff
@@ -43,4 +43,28 @@ is $validator->get('/name/$ref'), undef,    'get /name/$ref';
 is $validator->schema->get('/name/type'), 'string',             'schema get /name/type';
 is $validator->schema->get('/name/$ref'), '#/definitions/name', 'schema get /name/$ref';
 
+$bundled = $validator->schema('data://main/api.json')->bundle;
+is_deeply [ sort keys %{ $bundled->{definitions} } ], [ 'objtype' ], 'no dup definitions';
```

Comment:
Could you add this test to the other schemas above as well? I also think we should write a test where we load a schema from disk, with `$ref` to another file on disk. Like this: https://github.com/jhthorsen/json-validator/blob/master/t/deep-mixed-ref.t

Reply:
The other cases in `t/bundle.t` by definition (ha!) already work, see especially the line above here - the `$ref` is proved to be correct. The first bit of the file also can't behave differently because the `$fqn` will be empty-string.

I'll add similar testing to `t/deep-mixed-ref.t` right now.

Reply:
Done, see new commit.